pandas/_typing.py

```diff
@@ -28,6 +28,7 @@
 Scalar = Union[str, int, float]
 Axis = Union[str, int]
 Ordered = Optional[bool]
+Axes = Iterable[Union["Index", Iterable[str]]]
```

Comment:
Can you use `Collection[Hashable]` on the rhs here instead?

Reply:
Axes = Iterable[Union["Index", Collection[Hashable]]]

or

    Axes = Collection[Hashable]

I guess the first one right.

Reply:
`typing.Collection` was added in 3.6.0 I guess we are still supporting python 3.5 so I will revert the changes.

Reply:
@WillAyd do we have `typing-extensions` in dependency? Then we can use `Collections` from there.

Reply:
Hmm OK. We will actually be dropping Py35 support by the time this goes live. For now just go ahead and use Iterable and add a comment to use Container when Py35 gets dropped

Reply:
Alright.